kafka-ui-react-app/src/components/Brokers/Broker/Configs/Configs.tsx

```diff
@@ -34,14 +34,12 @@ const Configs: React.FC = () => {
 
   const getData = () => {
     return data
-      .filter(
-        (item) =>
-          item.name.toLocaleLowerCase().indexOf(keyword.toLocaleLowerCase()) >
-          -1
-      )
+      .filter((item) => {
+        const nameMatch = item.name.toLocaleLowerCase().includes(keyword);
+        return nameMatch ? true : item.value && item.value.toLocaleLowerCase().includes(keyword); // try to match the keyword on any of the item.value elements when nameMatch fails but item.value exists
```

Comment:
please cast the keyword to lowercase as well, otherwise it's not a case-insensitive search :)

Reply:
Yes @Haarolean. I have updated it.